main/src/cgeo/geocaching/connector/AbstractLogin.java

```diff
@@ -94,4 +96,9 @@ public StatusCode login(@Nullable final Credentials credentials) {
     @NonNull
     protected abstract StatusCode login(final boolean retry, @NonNull final Credentials credentials);
 
+    public void increaseActualCachesFound() {
+        if (actualCachesFound >= 0) {
+            setActualCachesFound(actualCachesFound + 1);
```

Comment:
That's actually `actualCachesFound++;`. Sorry, I though you already had this in mind when you proposed to reference the field directly.

Reply:
ah, sure